extras/src/test/java/com/google/gson/graph/GraphAdapterBuilderTest.java

```diff
@@ -66,6 +74,8 @@ public void testDeserialization() {
     assertSame(rock, paper.beats);
   }
 
+  @Ignore("Does not work anymore due to ReflectiveTypeAdapterFactory ignoring self references")
```

Comment:
That would work as well. Should I revert the usage of `@Test` in this test class and switch back to extending `TestCase` then? I mainly did it to be able to use `@Ignore`. Though eventually it might make sense to upgrade to JUnit 5 which would require switching to annotation based test methods.

Reply:
I think it is fine to keep the JUnit 4 migration.